main.go

```diff
@@ -79,7 +79,7 @@ func runAndLog(cmd *command.Model) (string, error) {
 	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
 	log.Donef(fmt.Sprintf("$ %s", cmd.PrintableCommandArgs()))
 	if err != nil {
-		return out, fmt.Errorf("error running npm command: %s", err)
+		return out, fmt.Errorf("error running npm command: %s", out)
```

Comment:
This pattern was recommended so far:
	
if out, err := model.RunAndReturnTrimmedCombinedOutput(); err != nil {
		if errorutil.IsExitStatusError(err) {
			failf("x failed: %s", out)
		} else {
			failf("x failed: %s", err)
		}
}